browser/app/js/objects/selectors.js

```diff
@@ -21,3 +21,12 @@ export const getCurrentPrefix = state => state.objects.currentPrefix
 export const getCheckedList = state => state.objects.checkedList
 
 export const getPrefixWritable = state => state.objects.prefixWritable
+
+const objectsSelector = state => state.objects.list
+const objectsFilterSelector = state => state.objects.filter
+
+export const getFilteredObjects = createSelector(
+  objectsSelector,
+  objectsFilterSelector,
+  (objects, filter) => objects.filter(object => object.name.startsWith(filter))
```

Comment:
Hi @saurabh29789, thank you very much for the contribution! Would it be possible to ignore case while searching for an object? It might be more intuitive for the user if we return the results ignoring the case.

When I have the following objects, and when I type `tes`, I don't get any search results (I would need to type `Tes` to get the search results):

- `Testobject1`
- `Testobject2`
- `otherobject`

To fix this, we can do something similar to the following:
```diff
diff --git a/browser/app/js/objects/selectors.js b/browser/app/js/objects/selectors.js
index 5a5439edb..5ad7ae026 100644
--- a/browser/app/js/objects/selectors.js
+++ b/browser/app/js/objects/selectors.js
@@ -28,5 +28,5 @@ const objectsFilterSelector = state => state.objects.filter
 export const getFilteredObjects = createSelector(
   objectsSelector,
   objectsFilterSelector,
-  (objects, filter) => objects.filter(object => object.name.startsWith(filter))
+  (objects, filter) => objects.filter(object => object.name.toLowerCase().startsWith(filter.toLowerCase()))
 )
```

Reply:
Good idea @kaankabalak since this would satisfy windows users as well

Reply:
@kaankabalak Sure, makes sense. Should we do this for bucket search as well ? I understand bucket names are lowercase only, but a user could type in a capitalized search query. Do we need to handle that case ?

Reply:
@saurabh29789 yes, we can do that as well